js-classes-inheritance/script.js

```diff
@@ -0,0 +1,94 @@
+class Builder {
+    constructor (arg) {
+        this.arg = arg;
+    }
+
+    plus(...n) {
```

Comment:
n it's an array (you can log it out:)) so you can iterate through it instead of using arguments